src/sentry/tasks/collect_project_platforms.py

```diff
@@ -37,5 +31,32 @@ def collect_project_platforms(**kwargs):
             )
         min_project_id += step
 
+
+@instrumented_task(name="sentry.tasks.collect_project_platforms", queue="stats")
+def collect_project_platforms(**kwargs):
+    now = timezone.now()
+
+    pre_snowflake_max_value = 2147483647
```

Comment:
This would be better as a constant that lives in the snowflake module, and we can import it here.

Also, it would be more readable as `(1 << 31) - 1`.